src/variational/advi.jl

```diff
@@ -34,101 +91,125 @@ function vi(model::Model, alg::ADVI; optimizer = TruncatedADAGrad())
         idx += varinfo.metadata[sym].ranges[end][end]
     end
 
-    q = Variational.MeanField(zeros(num_params), zeros(num_params), dists, ranges)
-    
-    # construct objective
-    elbo = ELBO()
+    # initial params
+    μ = randn(num_params)
+    σ = softplus.(randn(num_params))
```

Comment:
Shall we pass `rng` here somehow?

Reply:
Well, actually, is it necessary? This mainly just to initialze the model. If you want to ensure that reproducibility, you should just be instantiating the `MvNormal` yourself and use `bijector(model)` to get the model.

Reply:
Would it require users to use Bijectors.jl directly then? I think it's still nice if we allow user to pass RNGs via a keyword arguments. We just need to add `rng=GLOBAL_RNG` and pass it to the `randn`

Reply:
Nah, you still just do `Turing.Variational.bijector` right?
But yeah, we could just add it as a keyword argument if you think that's the best.

Reply:
What I mean is that users need to learn how to use the interface from Bijectors.jl, that contains defining the distribution, how `inv` and `Stacked` works and finally transformation, as you did in Line 98 to Line 103 next. Let's make a keyword argument possible, which is more friendly to end-users.

Reply:
Done!

Reply:
Cool thanks!